front/app/containers/Admin/projects/project/information/ReportSection.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+// api
+import usePhase from 'api/phases/usePhase';
+import useAddReport from 'api/reports/useAddReport';
+import useReport from 'api/reports/useReport';
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+// components
+import { SectionField, SubSectionTitle } from 'components/admin/Section';
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import ReportRow from './ReportRow';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+interface Props {
+  phaseId: string;
+}
+
+const ReportSection = ({ phaseId }: Props) => {
+  const { data: phase } = usePhase(phaseId);
+  const { mutate: createReport, isLoading } = useAddReport();
+  const phaseReportsEnabled = useFeatureFlag({ name: 'phase_reports' });
+
+  const reportId = phase?.data.relationships.report?.data?.id;
+  const { data: report } = useReport(reportId);
+
+  if (phase === undefined || !phaseReportsEnabled) return null;
```

Comment:
Should it specifically be `undefined` or it can be `null` too?
```suggestion
  if (isNil(phase) || !phaseReportsEnabled) return null;
```

Reply:
It can only be `undefined`